src/provider/view/utils/Markdown.ts

```diff
@@ -0,0 +1,16 @@
+const markdown = require('markdown-it');
```

Comment:
Can you do as:

    import * as markdown from 'markdown-it';

    const md = markdown.default('zero', {breaks: true});

so this is all properly typed?

Reply:
The reason I was using `require `here and in `Time.ts` was because the tests used `esModuleInterop `which would cause them to fail when importing either of those 2 files. I removed `esModuleInterop `from the test `tsconfig` and updated the `puppeteer` import so it works now.